en/theme/material/partials/toc.html

```diff
@@ -1,12 +1,29 @@
+<!--
```

Comment:
If this file is from the material theme we can't add our license header. Please verify it.